code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -29,17 +29,19 @@
 	if (istype(M,/mob/living/carbon/human))		//Repairing robolimbs
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/external/S = H.get_organ(user.zone_sel.selecting)
-
+		//VOREStation Edit Start
 		if (S && (S.robotic >= ORGAN_ROBOT))
 			if(!S.get_damage())
-				user << "<span class='notice'>Nothing to fix here.</span>"
+				to_chat(user, "<span class='notice'>Nothing to fix here.</span>")
+			else if((S.open < 2) && (S.get_damage() > 10) && !repair_external)
+				to_chat(user, "<span class='notice'>The damage is too extensive for this nanite swarm to handle.</span>")
 			else if(can_use(1))
 				user.setClickCooldown(user.get_attack_speed(src))
 				if(S.open >= 2)
 					if(do_after(user,5 * toolspeed))
 						S.heal_damage(20, 20, robo_repair = 1)
 				else if(do_after(user,5 * toolspeed))
-					S.heal_damage(10,10, robo_repair =1)
+					S.heal_damage(5,5, robo_repair =1)
```

Comment:
Can this be exported to a var? Could have it so that the advanced model still repairs 10.

Reply:
Okie